nemo_skills/dataset/mmlu_pro/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
let's rename folder to "mmlu-pro" for consistency with other datasets

Reply:
ok